packages/landing/ui/text/HeadingText.tsx

```diff
@@ -6,10 +6,10 @@ const headingTextStyles = cva([], {
     size: {
       6: ["text-[64px] leading-[72px]"],
       5: ["text-[52px] leading-[64px]"],
-      4: ["text-[40px] leading-[56px]"],
-      3: ["text-[32px] leading-[40px]"],
-      2: ["text-[26px] leading-[40px]"],
-      1: ["text-[20px] leading-[32px]"],
+      4: ["text-[26px] leading-[40px] md:text-[40px] md:leading-[56px]"],
+      3: ["text-xl leading-[32px] md:text-[32px] md:leading-[40px]"],
+      2: ["text-[20px] leading-8 md:text-[26px] md:leading-[40px]"],
+      1: ["text-[18px] md:text-[20px] md:leading-[32px]"],
```

Comment:
Are we going to sync this with app/ui ? Also applies to other /ui update on landing.

Reply:
We'll have to do a proper strategy to have responsive fonts on our UI.

As we'll use that UI components on new Swapr liquidity as well. We need to find a solution that will fit all of the different apps.